nativelib/src/main/resources/scala-native/gc/immix/Synchronizer.c

```diff
@@ -1,5 +1,6 @@
 #if defined(SCALANATIVE_MULTITHREADING_ENABLED) && defined(SCALANATIVE_GC_IMMIX)
 
+#include "../constants.h"
```

Comment:
Missed this one? Did you check out the `clangd`? I use it with VSCode and the plugin and it compiles everything as you are working on it. Very helpful.

Reply:
> Missed this one? Did you check out the clangd?

Oh, I didn't have that enabled, thanks.
Although even after enabling the LSP, it didn't point out anything, so I might have something wrong 😅 

Fixed in [fb292e9](https://github.com/scala-native/scala-native/pull/4310/commits/fb292e9dbfca87be933542b19acbe87adc1c2aec)